src/requirements/filtered-all-courses.ts

```diff
@@ -78,7 +76,7 @@ const filteredCoursesPaths: string[] = sortByLeastRecentRosters(
  * Then reduce to one JSON object
 */
 const filteredAllCourses: { readonly [semester: string]: readonly Course[] } = (
-  filteredCoursesPaths.map(path => JSON.parse(readFileSync('src/requirements/' + path).toString()))
+  filteredCoursesPaths.map(path => JSON.parse(readFileSync('src/requirements/filtered_courses/' + path).toString()))
```

Comment:
I think we can make it read the jsons in the functions folder, so that we don't have to maintain two identical files for each semester.

Reply:
Yes, that's a good point. I will push the changes so that we only keep and read from course data jsons in `functions/filtered_courses`